### PR TITLE
Slash in collection name v2

### PR DESCRIPTION
**Pull request by jamesgorrie**
No more encoding, and disallowing slashes for now, we can figure out what we do with the delimiter later.

### CODE REVIEW

File: common-lib/src/main/scala/com/gu/mediaservice/lib/collections/CollectionsManager.scala [hunk 1]
```diff
@@ -38,4 +35,7 @@ object CollectionsManager {
         col.path == collection.path && col.actionData.date.isAfter(collection.actionData.date)
       }}
     }
+
+  // We could use `ValidationNel`s here, but that's overkill
+  def isValidPathBit(s: String) = if (s.contains(delimiter)) false else true
```
Comment (theefer) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/collections/CollectionsManager.scala]: Did you mean `!`?

**Comment by theefer** [comment]
:+1:

File: common-lib/src/main/scala/com/gu/mediaservice/lib/collections/CollectionsManager.scala (hunk 1 above)
Comment (jamesgorrie) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/collections/CollectionsManager.scala]: Nope, if it contains the delimiter, it's invalid.

Comment (theefer) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/collections/CollectionsManager.scala]: `= ! s.contains(delimiter))`?

**Comment by paperboyo** [comment]
Oh, cool! What about `,` that is used to separate multiple labels, `~` if it stays for collections (I like it) and `#`?

**Pull request merged**